engine/src/main/java/org/terasology/engine/GameEngine.java

```diff
@@ -28,6 +28,23 @@
 public interface GameEngine extends AutoCloseable {
 
     /**
+     * @return The current, fine-grained status of the engine.
+     */
+    EngineStatus getStatus();
```

Comment:
I was just thinking that perhaps handling the game state here now felt a little out of place.  I'm now curious where you are gonna put it but I'll hold my tongue and wait (im)patiently...=D

Ultimately, ok for the interim confusion. =)